jupyter/environment-r-omero.yml

```diff
@@ -5,21 +5,21 @@ channels:
 - conda-forge
 - defaults
 dependencies:
-- bioconductor-ebimage
-- openjdk=8.0.*
-- r-assertthat
-- r-base=3.5.*
-- r-devtools
+- bioconductor-ebimage=4.24
+- openjdk=11
+- r-assertthat=0.2
+- r-base=3.5
```

Comment:
I think we could actually move to R 3.6 to be up to date. I'll give it a try.

Reply:
I noticed that we are using R 3.6 for the other dockerfile since we are using the latest docker image and that works

Reply:
Just tried myself, using 3.6 introduces lots of other version conflicts. Lets leave 3.5 for now 👍